libraries/testing/tester.cpp

```diff
@@ -950,6 +950,13 @@ namespace eosio { namespace testing {
       push_transaction( trx );
    }
 
+   bool base_tester::is_code_cached( eosio::chain::account_name name ) const {
+      const auto& db  = control->db();
+      const account_metadata_object* receiver_account = &db.template get<account_metadata_object,by_name>( name );
+      if ( receiver_account->code_hash == digest_type() ) return false;
```

Comment:
Do we need to check `receiver_account != nullptr` first? If the account doesn't exist

Reply:
`get()` throws unknown key if not found.

Reply:
oops yeah I was looking at `find()`